src/app/dynamic-forms/fields/cinchy-form-link.directive.ts

```diff
@@ -163,7 +163,7 @@ export class LinkDirective implements OnInit {
   }
 
   ngOnInit(): void {
-    this.showImage = this.field.cinchyColumn.dataFormatType === 'ImageUrl';
+    this.showImage = this.field.cinchyColumn.dataFormatType === 'ImageUrl(small)' || this.field.cinchyColumn.dataFormatType === 'ImageUrl(medium)' || this.field.cinchyColumn.dataFormatType === 'ImageUrl(large)';
```

Comment:
You could change these data types to be constants in a file. Also please change the values to have a space between the ImageUrl and the first parameter (eg. "ImageUrl(medium)" is "ImageUrl (medium)")